lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -398,6 +398,18 @@ getLocationInfoAsString(const SmallPtrSet<Operation *, 8> &ops) {
 //===----------------------------------------------------------------------===//
 
 namespace {
+struct ModuleEmitterOptions {
+  /// True if always_ff blocks should not be emitted.
+  bool noAlwaysFF = false;
```

Comment:
The double negative is hurting my head.  It makes sense for the attr name (since lack of attribute is use default, and yes_alwaysff is default), but should this struct have the positive version?  bool useAlwaysFF?

Reply:
+1 for `bool useAlwaysFF = true;`